apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedDevelopCore.md

```diff
@@ -0,0 +1,51 @@
+### Fabric Core
+
+[Fabric Core](https://github.com/OfficeDev/office-ui-fabric-core) is a collection of CSS classes and Sass mixins that give you access to Fabric's colors, animations, fonts, icons and grid.
+
+### Adding Fabric Core to your site
+
+To add Fabric Core to your site, link to this CSS file in the `<head>` of your webpage:
+
+```html
+<link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/9.6.1/css/fabric.min.css" />
+```
+
+You can also download a copy of Fabric for your project or add it through a package manager. You can also build your own copy from the source code.
+
+Add the ms-Fabric class to a containing element, such as `<body>`, to set the font-family for all Fabric typography classes used within that element. Then apply the Fabric styles directly to your HTML elements.
```

Comment:
```suggestion
Add the `.ms-Fabric` class to a containing element, such as `<body>`, to set the font-family for all Fabric typography classes used within that element. Then apply the Fabric styles directly to your HTML elements.
```

Reply:
I like the backticks but I'm not sure the dot is a good idea in this case, in case someone takes it literally (`<body class=".ms-Fabric">`)

Reply:
Had the same concern, seems reasonable to remove the `.`. Folks will copy and paste from the `<pre>` tag for sure.